package.json

```diff
@@ -81,5 +82,11 @@
     "onlyBuiltDependencies": [
       "esbuild"
     ]
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.18.3",
+    "@toruslabs/eccrypto": "^1.1.8",
+    "ethers": "^5.6.8",
+    "vite": "^2.9.9"
```

Comment:
@OlremProj are these accidental additions? Ethers is already in the `packages/rainbowkit` peerDeps. Not sure what others are being used for, but lemme know!